ecaudit/src/test/java/com/ericsson/bss/cassandra/ecaudit/logger/TestChronicleAuditLoggerConfig.java

```diff
@@ -122,4 +124,61 @@ public void testValidMaxLogSize()
         assertThat(config.getLogPath().toString()).isEqualTo("/tmp");
         assertThat(config.getMaxLogSize()).isEqualTo(1024L);
     }
+
+    @Test
+    public void testDefaultFieldsConfig()
+    {
+        Map<String, String> options = ImmutableMap.of("log_dir", "/tmp");
+
+        ChronicleAuditLoggerConfig config = new ChronicleAuditLoggerConfig(options);
+
+        FieldSelector fields = config.getFields();
+        assertThat(fields.isSelected(Field.CLIENT_IP)).isTrue();
+        assertThat(fields.isSelected(Field.CLIENT_PORT)).isTrue();
+        assertThat(fields.isSelected(Field.COORDINATOR_IP)).isTrue();
+        assertThat(fields.isSelected(Field.USER)).isTrue();
+        assertThat(fields.isSelected(Field.BATCH_ID)).isTrue();
+        assertThat(fields.isSelected(Field.STATUS)).isTrue();
+        assertThat(fields.isSelected(Field.OPERATION)).isTrue();
+        assertThat(fields.isSelected(Field.OPERATION_NAKED)).isFalse();
+        assertThat(fields.isSelected(Field.TIMESTAMP)).isTrue();
+    }
+
+    @Test
+    public void testCustomFieldsConfig()
+    {
+        Map<String, String> options = ImmutableMap.of("log_dir", "/tmp",
+                                                      "fields", ", CLIENT_IP, , USER, STATUS,");
+
+        ChronicleAuditLoggerConfig config = new ChronicleAuditLoggerConfig(options);
+
+        FieldSelector fields = config.getFields();
+        assertThat(fields.isSelected(Field.CLIENT_IP)).isTrue();
+        assertThat(fields.isSelected(Field.CLIENT_PORT)).isFalse();
+        assertThat(fields.isSelected(Field.COORDINATOR_IP)).isFalse();
+        assertThat(fields.isSelected(Field.USER)).isTrue();
+        assertThat(fields.isSelected(Field.BATCH_ID)).isFalse();
+        assertThat(fields.isSelected(Field.STATUS)).isTrue();
+        assertThat(fields.isSelected(Field.OPERATION)).isFalse();
+        assertThat(fields.isSelected(Field.OPERATION_NAKED)).isFalse();
+        assertThat(fields.isSelected(Field.TIMESTAMP)).isFalse();
+    }
+
+    @Test
+    public void testInvalidFieldsCycle()
+    {
+        Map<String, String> options = ImmutableMap.of("log_dir", "/tmp",
+                                                      "fields", "ErrorZ");
+
+        assertThatExceptionOfType(ConfigurationException.class)
+        .isThrownBy(() -> new ChronicleAuditLoggerConfig(options))
+        .withMessageContaining("fields")
+        .withMessageContaining("ErrorZ");
+    }
+
+    @Test
+    public void testDefaultFields()
```

Comment:
As this test case only tests the FieldSelector, should we test it in FieldSelector instead (already tested I think)?

Alternatively we could merge it with testDefaultFieldsConfig() and add a check that the default config produces this bitmap.

Reply:
True, this test can be removed. To start with this was not a constant defined in FieldSelector... After some refactoring this test was left here... But I'll remove it from here!